scripts/get

```diff
@@ -94,8 +94,54 @@ download_and_install() {
     curl -sSfL "$TARBALL_URL" | tar xfz - --strip-components=1 -C "$TMPDIR"
 
     echo Installing CRI-O
-    make -C "$TMPDIR"
+    pushd "$TMPDIR"
+
+    PREFIX=/usr/local
+    ETCDIR=/etc
+    CONTAINERS_DIR=$ETCDIR/containers
+    CNIDIR=$ETCDIR/cni/net.d
+    BINDIR=$PREFIX/bin
+    MANDIR=$PREFIX/share/man
+    OCIDIR=$PREFIX/share/oci-umount/oci-umount.d
+    SELINUX=$(selinuxenabled 2>/dev/null && echo -Z)
+    BASHINSTALLDIR=$PREFIX/share/bash-completion/completions
+    FISHINSTALLDIR=$PREFIX/share/fish/completions
+    ZSHINSTALLDIR=$PREFIX/share/zsh/site-functions
+    SYSTEMDDIR=$PREFIX/lib/systemd/system
+    OPT_CNI_BIN_DIR=/opt/cni/bin
+
+    install "$SELINUX" -d -m 755 $CNIDIR
```

Comment:
Thanks, I just verified `install` command exists on FCOS by default. It ships with very minimal set of commands, so I thought I will make sure anyway. 

```bash
[root@localhost core]# which install 
/usr/bin/install
```